### PR TITLE
GA pinned and pinned apis and event messages

### PR DESCRIPTION
**Pull request by sumanac**

### CODE REVIEW

File: api-reference/v1.0/toc.yml [hunk 1]
```diff
@@ -7359,6 +7365,15 @@ items:
...
           href: api/chatmessagehostedcontent-get.md
+      - name: Pinned chat message
+        href: resources/pinnedChatMessageInfo.md
+        items:
+        - name: List pinned messages
+          href: api/chat-list-pinnedmessages.md
+        - name: Pin message
+          href: api/chat-post-pinnedmessages.md
+        - name: Unpin message
+          href: api/chat-delete-pinnedmessages.md
```
Comment (RamjotSingh) [inline api-reference/v1.0/toc.yml:7376]: Let's add these under chat and under chat message as well

Comment (RamjotSingh) [reply api-reference/v1.0/toc.yml:7376]: Do the same for beta as well

**Review by RamjotSingh — CHANGES_REQUESTED** [review]

File: api-reference/v1.0/toc.yml (hunk 1 above)
Comment (sumanac) [reply api-reference/v1.0/toc.yml:7382]: pinned message related changes are already under "Chat". I added for "Chat message" for both beta and v1.0

File: api-reference/v1.0/api/chat-delete-pinnedmessages.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+|Application | Chat.ReadWrite.All |
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+DELETE /chats/{chat-id}/pinnedMessages/{pinnedChatMessageInfoId}
```
Comment (RamjotSingh) [inline api-reference/v1.0/api/chat-delete-pinnedmessages.md:31]: {pinnedChatMessageId} everywhere.

**Review by RamjotSingh — APPROVED** [review]

**Comment by sumanac** [comment]
@Lauragra - Could you please help review and merge this PR?

**Comment by JarbasHorst** [comment]
Hello @sumanac, we will start the content review work for this PR on Monday.

File: api-reference/v1.0/api/chat-delete-pinnedmessages.md (hunk 2 above)
Comment (sumanac) [reply api-reference/v1.0/api/chat-delete-pinnedmessages.md]: updated in both beta and v1.0

File: changelog/Microsoft.Teams.Core.json [hunk 3]
```diff
@@ -5542,6 +5538,47 @@
...
       "SubArea": ""
     },
+    {
+      "ChangeList": [
+        {
+          "Id": "490c5bb1-a4a4-4fff-a6f7-7da31f3bda5f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "pinnedMessages",
+          "ChangeType": "Addition",
+          "Description": "Added the **pinnedMessages** relationship to the [chat](https://docs.microsoft.com/en-us/graph/api/resources/chat?view=graph-rest-1.0) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.Core.json:5548]: Please, add the Pinned messages section to the Methods table in the chat resource topic (v1.0).

File: api-reference/v1.0/api/chat-list-pinnedmessages.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
+---
+title: "List pinned messages in a chat"
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/chat-list-pinnedmessages.md:2]: For consistency, the correct title for this API topic should be "List + navigation property name". The current title breaks our titling guidelines. To keep your titling style while also partially complaining with our guidelines, please update the title in lines 2 and 10 to be: 

- List pinnedChatMessage in a chat

Please, apply this change also to the corresponding beta topic.

**Comment by JarbasHorst** [comment]
Hello @sumanac, I reviewed your PR and left some comments. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [18424](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18424) into your branch to include my fixes and before addressing my new comments.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [18477](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18477) into this branch to include my final edits.

**Review by Lauragra — APPROVED** [review]

**Comment by sumanac** [comment]
@Jarbas-MSFT - Could you please merge this PR if there is no other changes needed?

**Comment by JarbasHorst** [comment]
Hi @sumanac, the content review work is completed for this PR. After the APIs are live, please set the ready to merge label to signal that your PR is ready to publish.

**Pull request merged**